Packs/ApiModules/ReleaseNotes/2_2_3.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### JSONFeedApiModule
+Fixed an issue when fetching indicators in Cortex XSOAR version below 6.5.0.
+##### HTTPFeedApiModule
+Fixed an issue when fetching indicators in Cortex XSOAR version below 6.5.0.
```

Comment:
```suggestion
Fixed an issue where fetching indicators in Cortex XSOAR with a version below 6.5.0 would fail.
##### HTTPFeedApiModule
Fixed an issue where fetching indicators in Cortex XSOAR with a version below 6.5.0 would fail.
```